src/App.jsx

```diff
@@ -1,20 +1,116 @@
-import "./App.css";
-import TodoList from "./TodoList";
-import AddTodoForm from "./AddTodoForm";
-import { useState } from "react";
+import React, { Fragment } from "react";
+import TodoList from "./components/TodoList";
+import AddTodoForm from "./components/AddTodoForm";
+import { useState, useEffect } from "react";
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  BrowserRouter,
+} from "react-router-dom";
+import styles from "./App.module.css";
 
 function App() {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
 
-  const [newTodo, setNewTodo] = useState("");
+  const query0 = "?view=Grid%20view";
```

Comment:
sort is not working with query1 and 2. I recommend changing the approach and instead doing a normal query with just query0, and then sorting the list you get. To add on this you can make some buttons to choose the sorting style and also optionally a use effect if youd like to have the sort apply to new additions to the list.